packages/template-retail-react-app/app/components/drawer-menu/index.jsx

```diff
@@ -0,0 +1,405 @@
+/*
```

Comment:
This looks to be a duplicate copy of what exists already in `drawer-menu/drawer-menu.jsx` 🤔 Perhaps it's accidentally copied.

Reply:
Yes there were a ton of changes from develop and some previous merges/reverted merges that complicated the merge process, but now everything should be sorted out.